submissions/andrewklmn/html-css-popup/index.html

```diff
@@ -8,113 +8,121 @@
 </head>
 <body>
   <nav>
-    <div class="topnav-right">
-      <a href="#">Mail</a>
-      <a href="#">Images</a>
-      <label class="btn show-items" for="popup-control">
-        <img class="icon" src="img/bento_icon.png" alt="Bento icon">
-      </label>
-      <a href="#">
-        <img class="icon" src="img/icon_bell.png" alt="Bell icon">
-      </a>
-      <a href="#">
-        <img class="icon" src="img/user_icon.png" alt="User icon">
-      </a>
-    </div>
-  </nav>
-  <div class="container">
-    <input class="popup-checkbox" type="checkbox" id="popup-control">
-    <div class="main-popup content">
-      <div class="menu-deco-square"></div>
-      <label class="fading-area-for-menu-closing" for="popup-control"></label>
-      <div class="auto-scrolled-area">
-        <div class="main-items-container content flex-items">
-          <a class="item" href="#">
-            <img src="img/popup-icons/calendar.png" alt="Calendar">
-              Calendar
-          </a>
-          <a class="item" href="#">
-              <img src="img/popup-icons/mail.png" alt="Mail">
-              Mail
-          </a>
-          <a class="item" href="#">
-            <img src="img/popup-icons/news.png" alt="News">
-             News
-          </a>
-
-          <a class="item" href="#">
-            <img src="img/popup-icons/maps.png" alt="Maps">
-              Map
-          </a>
-          <a class="item" href="#">
-              <img src="img/popup-icons/files.png" alt="Files">
-              Files
-          </a>
-          <a class="item" href="#">
-            <img src="img/popup-icons/contacts.png" alt="Contacts">
-             Contacts
-          </a>
-          <a class="item" href="#">
-            <img src="img/popup-icons/tanslation.png" alt="Translate">
-            Translate
-          </a>
-          <a class="item" href="#">
-            <img src="img/popup-icons/videos.png" alt="Videos">
-            Videos
-          </a>
-
-          <a class="item" href="#">
-          <img src="img/popup-icons/search.png" alt="Search">
-            Search
-          </a>
-          <a class="item" href="#">
-            <img src="img/popup-icons/photos.png" alt="Photo">
-              Photo
-          </a>
-          <a class="item" href="#">
-              <img src="img/popup-icons/apps.png" alt="Apps">
-              Apps
-          </a>
-          <a class="item" href="#">
-            <img src="img/popup-icons/documents.png" alt="Document">
-             Document
-          </a>
-
+    <ul class="topnav-right">
+      <li>
+        <a href="#" tabindex="1">Mail</a>
```

Comment:
By default all interactive elements get focus.
Not necessary to use `tabindex` for interactive elements if you don't want to change their order in markup.
[Read more about tabindex and how it works.](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/tabindex)